canonical-web-and-design/jaas.ai
# Add favicon to root

## Done

All modern browsers look in the site's root for a file named exactly favicon.ico, regardless of type, automatically.

Placing this icon in root will allow both the flask site and the dashboard to pick it up automatically.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- Verify the favicon appears as expected

## Details

Fixes #https://github.com/canonical-web-and-design/jaas-dashboard/issues/291
